src/features/artist/helpers/findArtistById.ts

```diff
@@ -0,0 +1,6 @@
+import { AlgoliaOfferWithArtistAndEan } from 'libs/algolia/types'
+
+export const findArtistById = (hits: AlgoliaOfferWithArtistAndEan[], artistId: string) =>
+  hits
+    .find((hit) => hit.artists?.some((a) => a?.id === artistId))
+    ?.artists?.find((a) => a?.id === artistId)
```

Comment:
```suggestion
  const relevantHit = hits.find((hit) => hit.artists?.some((a) => a?.id === artistId))
  return relevantHit.find((artist) => artist.id === artistId)
```

Je trouve ça plus lisible en 2 lignes (j'ai peut-être fait une erreur dans la suggestion)